ui/components/ResponsiveTable/index.js

```diff
@@ -115,13 +115,24 @@ export function ResponsiveTable({ schema, results }) {
         <Tbody>
           {results.map((result, index) => (
             <Tr key={index}>
-              {schema.map((s) => (
-                <Td key={s.id} title={s.title}>
-                  {s.formatter
-                    ? s.formatter(result[s.id], result)
-                    : result[s.id]}
-                </Td>
-              ))}
+              {schema.map((s) => {
+                const value = result[s.id];
+                return (
+                  <Td key={s.id} title={s.title}>
+                    {Array.isArray(value) && value.length > 0 ? (
+                      <ul className="nhsuk-list-bullet nhsuk-u-font-size-16">
+                        {value.map((val, index) => (
+                          <li key={index}>
+                            {val.formatter ? val.formatter(val, result) : val}
```

Comment:
Is it right that the location of the `formatter` changes depending on whether it's an array or not?

Why would this not _also_ be `s.formatter`?

Reply:
Modded to be the same (so my tired brain doesn't have to consider whether they're interchangeable in this instance)